proto/storage/compliance_operator_v2.proto

```diff
@@ -41,3 +49,31 @@ message ComplianceOperatorRuleV2 {
     string              rationale    = 10;
     string              fixes        = 11;
 }
+
+// Next Tag: 16
+message ComplianceOperatorScanSettingV2 {
+    string                        name                     = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    bool                          auto_apply_remediations  = 2;
+    bool                          auto_update_remediations = 3;
+    bool                          one_time_scan            = 4;
+    map<string, string>           labels                   = 5;
+    map<string, string>           annotations              = 6;
+    message Profile {
+        string                    profile_id               = 1 [(gogoproto.moretags) = 'search:"-" sql:"fk(ComplianceOperatorProfileV2:id),no-fk-constraint"'];
+    }
+    repeated Profile              profiles                 = 7;
+    message Cluster {
+        string                    cluster_id               = 1 [(gogoproto.moretags) = 'search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
+        string                    namespace                = 2 [(gogoproto.moretags) = 'search:"Namespace,store"'];
+        string                    namespace_id             = 3 [(gogoproto.moretags) = 'search:"Namespace ID" sql:"fk(NamespaceMetadata:id),no-fk-constraint,type(uuid)"'];
+    }
+    repeated Cluster              clusters                 = 8;
+    repeated NodeType             node_roles               = 9;
+    bool                          strict_node_scan         = 10;
+    // Starting point for schedule will probably have to build upon it
+    storage.Schedule              schedule                 = 11;
+    google.protobuf.Timestamp     created_time             = 12;
+    google.protobuf.Timestamp     last_updated_time        = 13;
+    google.protobuf.Timestamp     last_executed_time       = 14;
+    SlimUser                      user                     = 15;
```

Comment:
Is this a user who created or updated the settings?

Reply:
yes.  having created and updated users seemed a bit much.  figured just one would be sufficient.

Reply:
then my nit would be to rename the var to creator/created_by.